alert/types.go

```diff
@@ -154,3 +154,15 @@ type TopicState struct {
 	Level     Level
 	Collected int64
 }
+
+// AlertData is a structure that contains relevant data about an alert event.
+// The structure is intended to be JSON encoded, providing a consistent data format.
+type AlertData struct {
```

Comment:
The move was necessary since we moved the post handler out of the alertservice package correct?